README.md

```diff
@@ -1,11 +1,18 @@
-# nodejs-api-example
-Example nodejs code to integrate with the IMPARGO GraphQL API
+# IMPARGO API Nodejs Examples
+Example nodejs code to integrate with the IMPARGO GraphQL API. 
+As an introducting to the to API checkout the [Getting started guide](https://docs.google.com/document/d/1dl1iU7tzlj_vM0wvcWuWADaBNVYSvwHOlQGwP0-u1fE/edit?usp=sharing).
 
-## usage
+## Usage
 
 Run the following code to create a new order via the graphql api on the IMPARGO development system:
 ```sh
 yarn install
```

Comment:
I know we rely on yarn on all of our projects but maybe here it would be better to use npm to make a bit simpler / one less thing to install, what do you think? We would just need to replace the yarn.lock with a package-lock.json.

Reply:
Good point that makes sense it's probably less of a hurdle for some people. I switched to npm.